src/vmm/src/builder.rs

```diff
@@ -468,13 +468,34 @@ pub fn build_microvm_from_snapshot(
         }
     }
 
+    // Restore vcpus kvm state.
```

Comment:
nit: typo in the commit message: `VcupEvent::RestoreState` => `Vcpu::RestoreState`
```
threads. This in turn removed the need for `VcupEvent::RestoreState` and
```

Reply:
fixed